pkg/awsutils/awsutils.go

```diff
@@ -625,37 +621,40 @@ func (cache *EC2InstanceMetadataCache) createENI() (string, error) {
 }
 
 func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
-	tagSvc := cache.tagSVC
-	arns := make([]*string, 0)
-
-	eniARN := &arn.ARN{
-		Partition: "aws",
-		Service:   "ec2",
-		Region:    cache.region,
-		AccountID: cache.accountID,
-		Resource:  "network-interface/" + eniID}
-	arnString := eniARN.String()
-	arns = append(arns, aws.String(arnString))
+	tags := []*ec2.Tag{
+		{
+			Key:   aws.String(eniNodeTagKey),
+			Value: aws.String(cache.instanceID),
+		},
+	}
 
-	tags := make(map[string]*string)
+	clusterName := os.Getenv(clusterNameEnvVar)
+	if clusterName != "" {
+		tags = append(tags, &ec2.Tag{
+			Key:   aws.String(eniClusterTagKey),
+			Value: aws.String(clusterName),
+		})
+	}
 
-	tagValue := cache.instanceID
-	tags[eniTagKey] = aws.String(tagValue)
-	log.Debugf("Trying to tag newly created eni: keys=%s, value=%s", eniTagKey, tagValue)
+	for _, tag := range tags {
+		log.Debugf("Trying to tag newly created eni: key=%s, value=%s", *tag.Key, *tag.Value)
+	}
 
-	tagInput := &resourcegroupstaggingapi.TagResourcesInput{
-		ResourceARNList: arns,
-		Tags:            tags,
+	input := &ec2.CreateTagsInput{
+		Resources: []*string{
+			aws.String(eniID),
+		},
+		Tags: tags,
 	}
 
 	start := time.Now()
-	_, err := tagSvc.TagResources(tagInput)
-	awsAPILatency.WithLabelValues("TagResources", fmt.Sprint(err != nil)).Observe(msSince(start))
+	_, err := cache.ec2SVC.CreateTags(input)
+	awsAPILatency.WithLabelValues("CreateTags", fmt.Sprint(err != nil)).Observe(msSince(start))
 	if err != nil {
-		awsAPIErrInc("TagResources", err)
-		log.Warnf("Fail to tag the newly created eni %s  %v", eniID, err)
+		awsAPIErrInc("CreateTags", err)
+		log.Warnf("Fail to tag the newly created eni %s:  %v", eniID, err)
 	} else {
-		log.Debugf("Tag the newly created eni with arn: %s", arnString)
+		log.Debugf("Tag the newly created eni: %s", eniID)
```

Comment:
Can you please rephrase this message to
`"Successfully tagged eni: %s", eniID`

Reply:
Yup, makes more sense.